src/libraries/System.Private.CoreLib/src/System/Collections/Generic/List.cs

```diff
@@ -393,6 +393,31 @@ public void CopyTo(T[] array, int arrayIndex)
             Array.Copy(_items, 0, array, arrayIndex, _size);
         }
 
+        public void CopyTo(int sourceIndex, int count, Span<T> destination)
```

Comment:
@felipepessoto Your PR is adding new public APIs in the ref file. Please make sure to document them with triple slash comments so the reviewers can sign off your change. We want to make sure we always merge new public APIs with documentation.

Reply:
@carlossanlop, we are still adding this API?

Reply:
Thanks for documenting the public APIs with triple slash comments.
I'm not sure I understand the question. The purpose of this PR is to add these two APIs, so yes?